pkg/util/errors.go

```diff
@@ -0,0 +1,11 @@
+package util
```

Comment:
how about we move this into an `errutil` package in `pkg/util/errutil`?

I think its easier to understand
```go
return errutil.Wrap("failed to check if dashboard is provisioned", err)
```